### ISSUE TITLE
SNMP 64bit Counter Issue - Far from correct bandwidth values

### ISSUE THREAD

**Issue by GitStoph** [netdata/netdata#3488]
The values netdata is returning for some of my interfaces is about 15x higher than what the traffic is in reality. Could you help me determine why?
Config:
```
{
    "enable_autodetect": false,
    "update_every": 10,
    "max_request_size": 100,
    "servers": [
        {
             "hostname": "192.168.0.9",
             "community": "<oursnmpcommunity>",
             "update_every": 10,
             "max_request_size": 50,
             "options": { "timeout": 10000, "version": 1 },
             "charts": {
                "snmp_switch.HPStack_1.0.27": {
                    "title": "HP Stack to Charter",
                    "units": "kilobits/s",
                    "type": "area",
                    "priority": 100,
                    "family": "ISP",
                    "dimensions": {
                        "in": {
                            "oid": ".1.3.6.1.2.1.31.1.1.1.6.3",
                            "algorithm": "incremental",
                            "multiplier": 8,
                            "divisor": 1024,
                            "offset": 0
                        },
                        "out": {
                            "oid": ".1.3.6.1.2.1.31.1.1.1.10.3",
                            "algorithm": "incremental",
                            "multiplier": -8,
                            "divisor": 1024,
                            "offset": 0
                        }
                    }
                }
            }
        }
    ]
}
```
Using the following to debug:
```
# become user netdata
sudo su -s /bin/bash netdata
# run the SNMP plugin by hand
/usr/libexec/netdata/plugins.d/node.d.plugin debug 1 snmp
```
Debug output: https://pastebin.com/Bs2nQiYa

From the switch itself:
```
 Last 300 seconds input:  94410 packets/sec 119262856 bytes/sec 97%
 Last 300 seconds output:  9 packets/sec 633 bytes/sec 0%
```
It runs at this speed pretty consistently throughout the work day, so I would expect netdata to return values more along the lines of 900Mbps to 1Gbps. Instead however, netdata is returning:
![image](https://user-images.githubusercontent.com/20820226/36809017-62780dda-1c8c-11e8-84d0-6114cd72d241.png)

Our LibreNMS setup is correctly reporting the speed with the following values:
![image](https://user-images.githubusercontent.com/20820226/36809134-aba9519e-1c8c-11e8-8a91-6d2e67d5c786.png)

Thoughts as to why this may be? Did I do something wrong? Something you'd like me to try?
Thanks much!!

**Comment by GitStoph** [comment]
I changed `"divisor": 1024,` to `"divisor": 10240,` which obviously got me much closer to the true value I'd expect, but it's still hovering around 1.5Gbps.. does it not do the math to account for the difference over the 10seconds between data collections?

**Comment by cakrit** [comment]
This was fixed via PR #2633
